lib/loc.ml

```diff
@@ -36,12 +36,20 @@ let make_loc (bp, ep) = {
   bp = bp; ep = ep;
 }
 
+let is_dummy = function
+| { fname = ToplevelInput; line_nb = -1; bol_pos = 0; line_nb_last = -1; bol_pos_last = 0; bp = 0; ep = 0 } -> true
+| _ -> false
+
```

Comment:
We used to have that, however we removed it long time ago in favor of a stronger system using option types.

IMVHO re-adding this just because of an external plugin is not the best option, the plugin should be fixed instead.

Reply:
OK, the plugin does not seem too complex to fix. @mattam82, if I spend some time to change the dummy_loc into `None` there, is there some hope that this will be a useful work?

Reply:
I made mattam82/Coq-Equations/413 anyway. Let's see what @mattam82 says.